BLSQ/iaso
# POLIO-1091: use PrimaryKeyRelatedField for round

400 due to serializer error when changing polio round dates and saving

Related JIRA tickets :POLIO-1091

## Self proofreading checklist



## Changes

- Use `PrimaryKeyRelatedField` to serialize get the round pk.

## How to test

Go to polio campaigns, edit round dates:
- you should get a 200
- the dates should be correctly modified when you reopen the campaign
- the changes should be logged in the history

## Print screen / video


https://github.com/BLSQ/iaso/assets/38907762/94707ec6-2421-4788-9cba-88e684012983



## Notes

Relates to https://github.com/BLSQ/iaso/pull/702
